library/include/rocblas-functions.h

```diff
@@ -15350,7 +15350,7 @@ ROCBLAS_EXPORT const char* rocblas_status_to_string(rocblas_status status);
 /* \brief Initialize rocBLAS, to avoid costly startup time at the first call.
 */
 
-ROCBLAS_EXPORT void rocblas_init();
+ROCBLAS_EXPORT void rocblas_init(void);
```

Comment:
For functions declared int `rocblas-functions.h` or any other public headers which must compile with both C and C++, we should use `(void)` instead of `()` to indicate no arguments, because `(void)` is the same on C and C++, while `()` is old K&R syntax (before prototypes) in C.